app/src/main/java/com/jetbrains/handson/mpp/mobile/RecyclerViewCell.kt

```diff
@@ -0,0 +1,4 @@
+package com.jetbrains.handson.mpp.mobile
+
+data class RecyclerViewCell(val departureTime: String, val arrivalTime: String) {
+}
```

Comment:
I'm not sure this needs to be in its own file? Could be moved to RecyclerViewAdapter.kt because that's where it's relevant to?